packages/server/lib/automation/automation.ts

```diff
@@ -4,6 +4,7 @@ import { Cookies } from './cookies'
 import { Screenshot } from './screenshot'
 import type { BrowserPreRequest } from '@packages/proxy'
 import type { AutomationMiddleware, OnRequestEvent } from '@packages/types'
+import { removeAllCookies, removeCookie } from '../cookie-jar'
```

Comment:
without lookin into this deeply, why not merge the cookie jar and the cookies class? the names make it sound like one or the other could leverage each other and/or they could be merged?

Seems like the Cookies class might be able to.

Reply:
They serve different purposes. 

`lib/cookie-jar` is a stateful wrapper around tough-cookie's cookie implementation and stores cookies in memory for the sake of cross-origin testing. 

`lib/automation/cookies` is a stateless wrapper around automating cookies. It's more about taking actions with cookies than storing them.

I don't really see anything that can be shared between them and I think it makes sense to keep their concerns separate.

Reply:
hmm okay. Since the names are so similar, thoughts on updating both files to add a quick blur that describes their intent?

Reply:
Good idea. Added in [`ab6e987` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/ab6e98756701b1451c8d69dd6193cfbf0f0ac965)